### PR TITLE
UI Polish & TTL Update

### PR DESCRIPTION
**Pull request by vincentbavitz**
# MERGE #1242 FIRST

## Make the copy match mobile
image

## Leave Out "LNS Name"
- [x] On this screen, can we leave out the "or LNS name" for now as we haven't officially released that yet.
- [x] Also center it vertically in the text box
image

## Vertical Center Header
- [x] Make this vertically centered. It seems off when compared to the "Conversations" label and the "+" button.
image

## Temporarily Disable TTL Setting
If the user modifies the TTL that'll actually mess with push notifications on mobile
- [x] Set default TTL to 48 hours
- [x] Move TTL to `CONSTANTS`.

## Linking Device Copy
- [x] "Linking Device" rather than "Pairing Device"
- [x] "Scan the QR code on your other device" for consistency

### CODE REVIEW

**Comment by Mikunj** [comment]
We'll actually need to update the TTL of all `ContentMessage` children. We don't use `OutgoingMessage` anymore.

Also because all our TTL is now on ts, wouldn't it be better if we have the constants for the ttl set in ts too?

File: ts/session/messages/outgoing/content/ContentMessage.ts [hunk 1]
```diff
@@ -14,7 +14,6 @@ export abstract class ContentMessage extends Message {
    * this value can be used in all child classes
    */
   protected getDefaultTTL(): number {
-    // 1 day default for any other message
-    return 24 * 60 * 60 * 1000;
+    return window.CONSTANTS.TTL_DEFAULT_REGULAR_MESSAGE;
```
Comment (Mikunj) [inline ts/session/messages/outgoing/content/ContentMessage.ts]: can't we import this directly?
You'll also need to update `ttl()` in Pairing and device unpairing messages.

File: libtextsecure/outgoing_message.js [hunk 2]
```diff
@@ -13,14 +13,14 @@ const NUM_SEND_CONNECTIONS = 3;
 
 const getTTLForType = type => {
```
Comment (Bilb) [inline libtextsecure/outgoing_message.js:14]: this file is removed on clearnet. Be sure to remove those change so you do not have conflicts creating it back in.

File: ts/components/session/settings/SessionSettings.tsx [hunk 3]
```diff
@@ -441,7 +442,10 @@ export class SettingsView extends React.Component<SettingsViewProps, State> {
         id: 'message-ttl',
         title: window.i18n('messageTTL'),
         description: window.i18n('messageTTLSettingDescription'),
```
Comment (Mikunj) [inline ts/components/session/settings/SessionSettings.tsx:444]: just note that this setting is not used anywhere in our sending code at the moment and will need to be hooked up when we do enable it

**Pull request merged**